<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #DCE8C6</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonShowAll">
    <property name="geometry">
     <rect>
      <x>2000</x>
      <y>-40</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
        border-radius: 15px;
		border-right: 1px solid #aaaaaa;
		border-bottom: 1px solid #aaaaaa;
		font: bold 12pt &quot;Tahoma&quot;; /* Chữ in đậm */

}

QPushButton:enabled {
		background-color: #00806c;
		color: White;
}

QPushButton:pressed {
		background-color: #dfc456;
		color: #fffffe;
}


QPushButton:hover:!pressed {
		background-color: #86ad6f;
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Show All</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/statistics(1).png</normaloff>../images/statistics(1).png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1394</x>
      <y>-650</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-20</y>
      <width>1121</width>
      <height>891</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #DCE8C6</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1071</width>
      <height>871</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>60</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Total Employee</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>220</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Total Supplier</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSupplier">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>370</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonExit">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>730</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
        border-radius: 15px;
		border-right: 1px solid #aaaaaa;
		border-bottom: 1px solid #aaaaaa;
		font: bold 12pt &quot;Tahoma&quot;; /* Chữ in đậm */

}

QPushButton:enabled {
		background-color: #00806c;
		color: White;
}

QPushButton:pressed {
		background-color: #dfc456;
		color: #fffffe;
}


QPushButton:hover:!pressed {
		background-color: #86ad6f;
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../images/back.png</normaloff>../../images/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>100</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../images/division (2)(1).png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBarChart">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>660</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
        border-radius: 15px;
		border-right: 1px solid #aaaaaa;
		border-bottom: 1px solid #aaaaaa;
		font: bold 12pt &quot;Tahoma&quot;; /* Chữ in đậm */

}

QPushButton:enabled {
		background-color: #00806c;
		color: White;
}

QPushButton:pressed {
		background-color: #dfc456;
		color: #fffffe;
}


QPushButton:hover:!pressed {
		background-color: #86ad6f;
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>Bar Product</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/bar-chart(1).png</normaloff>../images/bar-chart(1).png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPieChart">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>660</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
        border-radius: 15px;
		border-right: 1px solid #aaaaaa;
		border-bottom: 1px solid #aaaaaa;
		font: bold 12pt &quot;Tahoma&quot;; /* Chữ in đậm */

}

QPushButton:enabled {
		background-color: #00806c;
		color: White;
}

QPushButton:pressed {
		background-color: #dfc456;
		color: #fffffe;
}


QPushButton:hover:!pressed {
		background-color: #86ad6f;
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>Pie Category</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/pie-chart(1).png</normaloff>../images/pie-chart(1).png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>420</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Total Category</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>670</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../images/products(1).png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>280</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../images/manufacture(1).png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>470</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../images/market-segment(1).png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="labelEmployee">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>170</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCategory">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>550</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDistribution">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>730</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
        border-radius: 15px;
		border-right: 1px solid #aaaaaa;
		border-bottom: 1px solid #aaaaaa;
		font: bold 12pt &quot;Tahoma&quot;; /* Chữ in đậm */

}

QPushButton:enabled {
		background-color: #00806c;
		color: White;
}

QPushButton:pressed {
		background-color: #dfc456;
		color: #fffffe;
}


QPushButton:hover:!pressed {
		background-color: #86ad6f;
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>Distribution</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/distribution(1).png</normaloff>../images/distribution(1).png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>610</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Total Product</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLineChart">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>660</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
        border-radius: 15px;
		border-right: 1px solid #aaaaaa;
		border-bottom: 1px solid #aaaaaa;
		font: bold 12pt &quot;Tahoma&quot;; /* Chữ in đậm */

}

QPushButton:enabled {
		background-color: #00806c;
		color: White;
}

QPushButton:pressed {
		background-color: #dfc456;
		color: #fffffe;
}


QPushButton:hover:!pressed {
		background-color: #86ad6f;
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>Line Product</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/line-chart(1).png</normaloff>../images/line-chart(1).png</iconset>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>831</width>
       <height>591</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
    <widget class="QLabel" name="labelProduct">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>710</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPieShift">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>730</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
        border-radius: 15px;
		border-right: 1px solid #aaaaaa;
		border-bottom: 1px solid #aaaaaa;
		font: bold 12pt &quot;Tahoma&quot;; /* Chữ in đậm */

}

QPushButton:enabled {
		background-color: #00806c;
		color: White;
}

QPushButton:pressed {
		background-color: #dfc456;
		color: #fffffe;
}


QPushButton:hover:!pressed {
		background-color: #86ad6f;
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>Pie Shift</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/replacement(1).png</normaloff>../images/replacement(1).png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonShowAll_2">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>0</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
        border-radius: 15px;
		border-right: 1px solid #aaaaaa;
		border-bottom: 1px solid #aaaaaa;
		font: bold 12pt &quot;Tahoma&quot;; /* Chữ in đậm */

}

QPushButton:enabled {
		background-color: #00806c;
		color: White;
}

QPushButton:pressed {
		background-color: #dfc456;
		color: #fffffe;
}


QPushButton:hover:!pressed {
		background-color: #86ad6f;
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>Show All</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/statistics(1).png</normaloff>../images/statistics(1).png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>431</width>
       <height>48</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-size: 40px;
    font-weight: bold;
	color: rgb(0, 128, 108);
}
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Visualization&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
